### PR TITLE
build: adds PortfolioStateZero component

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Adds portfolio state zero page
- Adds welcome to portfolio nav
- Updates toggle component to match new design
- Updates links to portfolio page to go to welcome or overview depending on a account being connected and if there is a balance

### CODE REVIEW

File: packages/shared/notionable-hooks/src/use-account.ts [hunk 1]
```diff
@@ -48,6 +48,16 @@ export function useAccountReady(network: Network | undefined) {
   return useAccountDefinition(network) !== undefined;
 }
 
+
+export function useAcctAndBalanceReady(network: Network | undefined) {
```
Comment (jeffywu) [inline packages/shared/notionable-hooks/src/use-account.ts]: Can you spell out Acct here? Thats the convention I'm using elsewhere.

File: packages/shared/notionable-hooks/src/use-account.ts [hunk 2]
```diff
@@ -48,6 +48,16 @@ export function useAccountReady(network: Network | undefined) {
...
+  const isAccountReady = useAccountReady(network);
+  const walletBalances = useAccountNetWorth();
```
Comment (jeffywu) [inline packages/shared/notionable-hooks/src/use-account.ts]: This variable should be called `accountNetWorth`

File: packages/shared/web/src/header/use-nav-links.tsx [hunk 3]
```diff
@@ -9,16 +9,28 @@ import {
...
 import InvestAndEarnDropdown from './invest-and-earn/invest-and-earn-dropdown/invest-and-earn-dropdown';
 import BorrowDropDown from './borrow-dropdown/borrow-dropdown';
+import {
+  useAccountNetWorth,
+  useAccountReady,
+} from '@notional-finance/notionable-hooks';
 
 export const useNavLinks = (mobileNav: boolean, theme: NotionalTheme) => {
   const network = useDefaultNetwork();
+  const isAccountReady = useAccountReady(network);
```
Comment (jeffywu) [inline packages/shared/web/src/header/use-nav-links.tsx]: Can't you use the other hook here instead?

File: packages/features/portfolio/src/containers/portfolio-state-zero/hooks/use-token-data.tsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+  const yieldData = selectedTabIndex === 2 ? borrowYields : earnYields;
+
+  const tokenObj: {
+    [symbol: string]: { products: string[]; data: typeof earnYields };
+  } = {};
+
+  const leveragedTokenData: Record<string, any> = {};
+
+  if (selectedTabIndex === 1) {
```
Comment (jeffywu) [inline packages/features/portfolio/src/containers/portfolio-state-zero/hooks/use-token-data.tsx]: Can you create named constants for the different tabs so these are easier to read?

Comment (matthew-garrett) [reply packages/features/portfolio/src/containers/portfolio-state-zero/hooks/use-token-data.tsx]: the mui tabs element needs numbers. I can create a object with names at the component level if you think that would help

Comment (jeffywu) [reply packages/features/portfolio/src/containers/portfolio-state-zero/hooks/use-token-data.tsx]: Yes thats what I mean

Comment (matthew-garrett) [reply packages/features/portfolio/src/containers/portfolio-state-zero/hooks/use-token-data.tsx]: Cool, lmk if you're happy with my solution

**Pull request merged**